<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchClass</class>
 <widget class="QMainWindow" name="searchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1780</width>
    <height>1266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>知乎日报搜索引擎</string>
  </property>
  <property name="windowIcon">
   <iconset resource="search.qrc">
    <normaloff>:/search/8c1001e93901213fc7ec80e957e736d12f2e952c.jpg</normaloff>:/search/8c1001e93901213fc7ec80e957e736d12f2e952c.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>知乎日报网页关键词搜索</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 满足最奇妙的好奇心，搜罗全球值得讨论的话题，由扎根世界各地的朋友们带给你最真实的当地情况</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>117</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>搜索</string>
            </property>
            <property name="icon">
             <iconset resource="search.qrc">
              <normaloff>:/search/8c1001e93901213fc7ec80e957e736d12f2e952c.jpg</normaloff>:/search/8c1001e93901213fc7ec80e957e736d12f2e952c.jpg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>使用说明：在搜索框中输入关键词，点击搜索，搜索结果将显示在下方框内，想要查看相关包含关键词的网页，请点击搜索结果即可</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="search.qrc"/>
 </resources>
 <connections/>
</ui>
